.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,80 @@
+name: Bug Report
+about: Report an issue related to the Home Assistant Supervisor.
+labels: bug
+title: ""
+issue_body: true
+inputs:
+  - type: description
+    attributes:
+      value: |
+        This issue form is for reporting bugs with **supported** setups only!
+
+        If you have a feature or enhancement request, please use the [feature request][fr] section of our [Community Forum][fr].
+        [fr]: https://community.home-assistant.io/c/feature-requests
+  - type: input
+    attributes:
+      label: What is the version of the Supervisor used?
+      required: true
+      placeholder: supervisor-
+      description: >
+        Can be found in the Supervisor panel -> System tab. Starts with
+        `supervisor-....`.
+  - type: dropdown
+    attributes:
+      label: What type of installation are you running?
+      required: true
+      description: >
+        If you don't know, you can find it in: Configuration panel -> Info.
+      choices:
+        - Home Assistant OS
+        - Home Assistant Supervised
+  - type: dropdown
+    attributes:
+      label: Which operating system are you running on?
+      required: true
+      choices:
+        - Home Assistant Operating System (HassOS)
+        - Debian
+        - Other (e.g., Raspbian/Raspberry Pi OS/Fedora)
```

Comment:
```suggestion
```
You are instructed at the beginning of the file to not use this for unsupported, I don't think this is needed.

Reply:
I actually added it on purpose. For 2 reasons:

1. People don't care about the comment.
2. It's a trap.
3. Please don't know Raspbian and Debian are different (Raspabian Debian (Buster) is part of their OS description).

By making this more explicit, it becomes more clear.

Reply:
Okey then :+1: